.eslintrc.js

```diff
@@ -221,7 +221,7 @@ module.exports = {
             'error',
             {
                 paths: [{ name: '.' }, { name: '..' }, { name: '../..' }],
-                patterns: ['**/libDev/src'],
+                patterns: ['**/libDev/src', '@trezor/utils/lib', '@trezor/utils/lib/*'],
```

Comment:
ok, so I understand this is a helper to forbit importing fro @trezor/utils and to prevent this 'antipattern' from now on. In the end, something like @trezor/**/lib will end up here?

Reply:
Exactly